pkg/metrics/metrics.go

```diff
@@ -32,7 +32,47 @@ var (
 	}, []string{moduleNameLabel, kymaNameLabel, stateLabel, shootIDLabel, instanceIDLabel})
 )
 
+//nolint:gomnd
 func Initialize() {
+	requestLatency := prometheus.NewHistogramVec(
```

Comment:
Please mention it's temporary solution, and should be removed after controller-runtime upgraded to 0.15.0